src/core/shards/flow.cpp

```diff
@@ -451,36 +451,36 @@ struct Maybe : public BaseSubFlow {
   }
 
   SHVar activate(SHContext *context, const SHVar &input) {
-    SHVar output{};
+    SHVar output = input;
```

Comment:
Does it clone or copy here?

Reply:
simple blit copy since it's basic type `SHVar`
It's just to fix the flow and make it consistent.